src/octoprint/__init__.py

```diff
@@ -184,6 +184,11 @@ def init_platform(
     if callable(after_environment_detector):
         after_environment_detector(**kwargs)
 
+    try:
+        init_systemcommands()
+    except Exception as ex:
+        raise FatalStartupError("Could not initialize system commands", cause=ex)
+
```

Comment:
This shouldn't get initialized here but rather only as needed. If adjusted as mentioned, there's no need for explicit initialization anyhow.